rskj-core/src/main/java/org/ethereum/core/Transaction.java

```diff
@@ -204,7 +205,11 @@ public long transactionCost(Constants constants, ActivationConfig.ForBlock activ
         long nonZeroes = this.nonZeroDataBytes();
         long zeroVals = ListArrayUtil.getLength(this.getData()) - nonZeroes;
 
-        return (this.isContractCreation() ? GasCost.TRANSACTION_CREATE_CONTRACT : GasCost.TRANSACTION) + zeroVals * GasCost.TX_ZERO_DATA + nonZeroes * GasCost.TX_NO_ZERO_DATA;
+        return (this.isContractCreation() ? GasCost.TRANSACTION_CREATE_CONTRACT : GasCost.TRANSACTION) + zeroVals * GasCost.TX_ZERO_DATA + nonZeroes * getTxNoZeroData(activations);
```

Comment:
Can we make this more readable?

A suggestions can be:
```
long transactionCost =  this.isContractCreation() ? GasCost.TRANSACTION_CREATE_CONTRACT : GasCost.TRANSACTION;

long transactionZeroData = getTxNoZeroData(activations);

long totalTransactionCost = transactionCost + zeroVals * GasCost.TX_ZERO_DATA + nonZeroes * transactionZeroData;

return totalTransactionCost;
```

I think this just to make it easier to read the calculation process.

Reply:
Done